src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/MessageType.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
```

Comment:
Could we create a unit test for `MessageType` please? 

Something like `WorkflowIdTest.java` or `PersonIdentificationTypeTest.java`